lib/beaker-hostgenerator/data.rb

```diff
@@ -29,6 +56,873 @@ def pe_upgrade_family
       'm' => 'master',
     }
 
+    BASE_CONFIG = {
+      'HOSTS' => {},
+      'CONFIG' => {
+        'nfs_server' => 'none',
+        'consoleport' => 443,
+      }
+    }
+
+    def base_host_config
+      {
+        'pe_dir' => pe_dir(pe_version, pe_family),
+        'pe_ver' => pe_version,
+        'pe_upgrade_dir' => pe_dir(pe_upgrade_version, pe_upgrade_family),
+        'pe_upgrade_ver' => pe_upgrade_version,
+      }
+    end
+
+    def osinfo
+      {
```

Comment:
Absolutely, and one place we might start sharing the OS info rather than duplicating it is in Beaker. 
@kevpl and @tvpartytonight and myself were floating that idea around, but nothing actionable yet.

I wanted to work towards that goal in this PR by consolidating all the info and decoupling it from the vmpooler hypvisor.